doc/en/reference.rst

```diff
@@ -1652,3 +1652,306 @@ passed multiple times. The expected format is ``name=value``. For example::
+      -V, --version         display pytest version and information about
+                            plugins.When given twice, also display information
+                            about plugins.
+      -h, --help            show help message and configuration info
+      -p name               early-load given plugin module name or entry point
+                            (multi-allowed).
+                            To avoid loading of plugins, use the `no:` prefix,
+                            e.g. `no:doctest`.
+      --trace-config        trace considerations of conftest.py files.
+      --debug               store internal tracing debug information in
+                            'pytestdebug.log'.
+      -o OVERRIDE_INI, --override-ini=OVERRIDE_INI
+                            override ini option with "option=value" style, e.g.
+                            `-o xfail_strict=True -o cache_dir=cache`.
+      --assert=MODE         Control assertion debugging tools.
+                            'plain' performs no assertion debugging.
+                            'rewrite' (the default) rewrites assert statements
+                            in test modules on import to provide assert
+                            expression information.
+      --setup-only          only setup fixtures, do not execute tests.
+      --setup-show          show setup of fixtures while executing tests.
+      --setup-plan          show what fixtures and tests would be executed but
+                            don't execute anything.
+
+    logging:
+      --log-level=LEVEL     level of messages to catch/display.
+                            Not set by default, so it depends on the root/parent
+                            log handler's effective level, where it is "WARNING"
+                            by default.
+      --log-format=LOG_FORMAT
+                            log format as used by the logging module.
+      --log-date-format=LOG_DATE_FORMAT
+                            log date format as used by the logging module.
+      --log-cli-level=LOG_CLI_LEVEL
+                            cli logging level.
+      --log-cli-format=LOG_CLI_FORMAT
+                            log format as used by the logging module.
+      --log-cli-date-format=LOG_CLI_DATE_FORMAT
+                            log date format as used by the logging module.
+      --log-file=LOG_FILE   path to a file when logging will be written to.
+      --log-file-level=LOG_FILE_LEVEL
+                            log file logging level.
+      --log-file-format=LOG_FILE_FORMAT
+                            log format as used by the logging module.
+      --log-file-date-format=LOG_FILE_DATE_FORMAT
+                            log date format as used by the logging module.
+      --log-auto-indent=LOG_AUTO_INDENT
+                            Auto-indent multiline messages passed to the logging
+                            module. Accepts true|on, false|off or an integer.
+
+    Hypothesis:
```

Comment:
this includes our plugins it looks like so we probably need to filter those out somehow?

Reply:
Ahh my bad. I ran regendoc manually. I will update this.